apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -1672,3 +1687,55 @@ type APIManagerList struct {
 func init() {
 	SchemeBuilder.Register(&APIManager{}, &APIManagerList{})
 }
+
+func (apimanager *APIManager) IsSystemRedisTLSEnabled() bool {
+	tlsEnabled := false
+	if apimanager.Spec.SystemRedisTLSEnabled != nil &&
+		*apimanager.Spec.SystemRedisTLSEnabled &&
+		apimanager.Spec.ExternalComponents != nil &&
```

Comment:
external components check should not be here. All installations of 3scale on 2.16 + must use external.

Reply:
Done. Thanks